config.md

```diff
@@ -90,7 +90,9 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 
 ## Process configuration
 
-* **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process, defaults to false.
+* **`terminal`** (bool, optional) specifies whether a terminal is attached to that process, defaults to false.
+  On Linux, a psuedoterminal pair is allocated for the container process and the psuedoterminal slave is duplicated on the container process's [standard streams][stdin.3].
+  Also, the psuedoterminal slave is bind mounted at /dev/console.
```

Comment:
I think “bind mounted” is overly specific.  Ccon does not bind mount `/dev/console`, it relies on the container's [`posix_openpt`][1] and the kernel/libc implementation to get the appropriate console device behavior.  In this case, I suggest we lead with the ocitools implementation of the test (how do you validate proper /dev/console handling?) and then figure out the wording after we have ocitools tests we like.  I expect the test will be something like “container writes to /dev/console and test harness ensures it reads that string out of the pseudoterminal master”, but we may want other tests in addition (I'm not familiar enough with /dev/console to know).

[1]: http://man7.org/linux/man-pages/man3/posix_openpt.3.html